### PR TITLE
[PromptGen] Basic implementation

### PR DESCRIPTION
**Pull request by DragonZoom**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,4 +31,7 @@
 *.out
 *.app
 
-/build
\ No newline at end of file
+/build
```
Comment (kilka-rav) [inline .gitignore:34]: Good!

File: AUTesting/PGenerator.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def refineFromError(self, error: str) -> "Prompt":
 
     def generate(self) -> str:
         prompt = """
-Your task is to generate a runnable test case code for the provided code (Use google tests). Please ensure that the test case covers all possible scenarios and edge cases, and that the code is easy to read and understand. Your response should only include the runnable code. Do not return any original code. Additionally, please make sure that the test case is well-organized and follows best practices for testing.
+Your task is to generate a runnable test case code for the provided code. Please ensure that the test case covers all possible scenarios and edge cases, and that the code is easy to read and understand. Your response should only include the runnable code. Do not return any original code. Additionally, please make sure that the test case is well-organized and follows best practices for testing.
 """
```
Comment (kilka-rav) [inline AUTesting/PGenerator.py:38]: Thanks for refactoring prompt. You are really good prompt-engineer🥰

File: main.py [hunk 3]
```diff
@@ -24,6 +24,7 @@ def extract_code_from_chatgpt_response(response):
 
     return cleaned_code_blocks
 
+
```
Comment (kilka-rav) [inline main.py:27]: Is it neccessary?🧐
